File: Serial.java
public class Serial {
	
	public static void main(String args[]) throws InterruptedException {
		Serial t = new Serial();
		System.out.println("Start!");
		long startTime = System.currentTimeMillis();
		t.testCase1();
		t.testCase2();
		t.testCase3();
		t.testCase4();
		long endTime = System.currentTimeMillis();
		long wait = ((endTime- startTime)/1000);
		System.out.println("End!");
		System.out.println("Time Taken : "+wait+" sec" );
	}
	
	public void testCase1() throws InterruptedException{
		Thread.sleep(1000);		
	}

	public void testCase2()throws InterruptedException {
		Thread.sleep(2000);
	}
	
	public void testCase3()throws InterruptedException {
		Thread.sleep(3000);
	}
	public void testCase4()throws InterruptedException {
		Thread.sleep(4000);
	}
}

File: Serial.py
import time
print("Start")
def Testcase1():
    time.sleep(1)
def Testcase2():
    time.sleep(2)
def Testcase3():
    time.sleep(3)
def Testcase4():
    time.sleep(4)
start_time = time.time()
Testcase1()
Testcase2()
Testcase3()
Testcase4()
end_time = time.time()
print("total time for all testcases are  ",round(end_time-start_time), "seconds", "\nDone")

